server/src/main/java/org/elasticsearch/index/stats/IndexingPressureStats.java

```diff
@@ -46,6 +46,13 @@
     private final long replicaRejections;
     private final long memoryLimit;
 
+    private final long totalCoordinatingOps;
```

Comment:
Perhaps add a comment on these that we intend to expose these or something similar in the future in the x-content after experimentation.